### PR TITLE
blog(embertimes#59): add Component Bounds RFC writeup

### PR DESCRIPTION
**Pull request by jessica-jordan**
## What it does

This adds a short writeup on the Component Bounds RFC: https://github.com/emberjs/rfcs/pull/351

## Sources

screen shot 2018-08-09 at 23 43 40

### CODE REVIEW

File: source/blog/2018-08-10-the-ember-times-issue-59.md [hunk 1]
```diff
@@ -31,8 +31,28 @@ You can already test this by [getting the beta](https://github.com/ember-cli/emb
...
 ---
 
-## [SECTION TITLE](#section-url)
-
+## [Knowing Boundaries with the Component Manager Bounds RFC](https://github.com/emberjs/rfcs/pull/351)
+
+A new Request for Comments (RFC) is aiming to improve the flexibility of another previously
+RFC'ed feature that is interesting for Ember addon authors who are passionate about `Components`.
+The [Component Manager™ feature](https://emberjs.github.io/rfcs/0213-custom-components.html) which had
+already had been accepted by the Ember community in the spring of this year ([xxx](https://github.com/emberjs/rfcs/pull/213#issuecomment-374981569)) defined a **new low-level primitive**
```
Comment (amyrlam) [inline source/blog/2018-08-10-the-ember-times-issue-59.md]: maybe not `xxx` just `emberjs/rfcs/pull/213` or something?

File: source/blog/2018-08-10-the-ember-times-issue-59.md [hunk 2]
```diff
@@ -31,8 +31,28 @@ You can already test this by [getting the beta](https://github.com/ember-cli/emb
...
+that would allow developers to create flexible and efficient **component base classes**
+which can easily be extended from. This can for example be leveraged by UI addons that expose
+components to their parent apps and which want to allow end users to extend from these base components themselves.
+
+And today the saga of the Manager of Custom Components continues:
+The author of the original RFC#213 now presents a [follow-up](https://github.com/emberjs/rfcs/pull/351)
```
Comment (amyrlam) [inline source/blog/2018-08-10-the-ember-times-issue-59.md]: maybe consider `RFC #213` with space?

File: source/blog/2018-08-10-the-ember-times-issue-59.md (hunk 1 above)
Comment (jessica-jordan) [reply source/blog/2018-08-10-the-ember-times-issue-59.md]: Totally, missed that one, good catch :+1:

File: source/blog/2018-08-10-the-ember-times-issue-59.md (hunk 2 above)
Comment (jessica-jordan) [reply source/blog/2018-08-10-the-ember-times-issue-59.md]: Yes, sounds great :+1:

**Comment by amyrlam** [comment]
merging, can make further edits on the main branch if needed

**Pull request merged**